modules/precisoBidAdapter.js

```diff
@@ -0,0 +1,513 @@
+import {deepAccess, isArray, logError, logInfo, mergeDeep} from '../src/utils.js';
+import {getOrigin} from '../libraries/getOrigin/index.js';
+import {BANNER, NATIVE, VIDEO} from '../src/mediaTypes.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {includes} from '../src/polyfill.js';
+import {convertOrtbRequestToProprietaryNative} from '../src/native.js';
+import {config} from '../src/config.js';
+
+const BIDDER_CODE = 'preciso';
+const REGIONS = ['prebid-eu', 'prebid-us', 'prebid-asia'];
+const ENDPOINT_URL = 'mndtrk.com/bidder/prebid/bids';
+const FLEDGE_ENDPOINT_URL = 'mndtrk.com/bidder/prebidfledge/bids';
+const DEFAULT_CURRENCY_ARR = ['USD']; // NOTE - USD is the only supported currency right now; Hardcoded for bids
+const SUPPORTED_MEDIA_TYPES = [BANNER, NATIVE, VIDEO];
+const TTL = 100;
+const GVLID = 874;
+
+// Codes defined by OpenRTB Native Ads 1.1 specification
+export const OPENRTB = {
+  NATIVE: {
+    IMAGE_TYPE: {
+      ICON: 1,
+      MAIN: 3,
+    },
+    ASSET_ID: {
+      TITLE: 1,
+      IMAGE: 2,
+      ICON: 3,
+      BODY: 4,
+      SPONSORED: 5,
+      CTA: 6
+    },
+    DATA_ASSET_TYPE: {
+      SPONSORED: 1,
+      DESC: 2,
+      CTA_TEXT: 12,
+    },
+  }
+};
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: SUPPORTED_MEDIA_TYPES,
+  gvlid: GVLID,
+
+  isBidRequestValid: function (bid) {
+    return !!(includes(REGIONS, bid.params.region) && bid.params.publisherId);
+  },
+  buildRequests: function (validBidRequests, bidderRequest) {
+    // convert Native ORTB definition to old-style prebid native definition
+    validBidRequests = convertOrtbRequestToProprietaryNative(validBidRequests);
+
+    const request = {
+      id: bidderRequest.bidderRequestId,
+      imp: validBidRequests.map(slot => mapImpression(slot, bidderRequest)),
+      site: mapSite(validBidRequests, bidderRequest),
+      cur: DEFAULT_CURRENCY_ARR,
+      test: validBidRequests[0].params.test || 0,
```

Comment:
Can you update the adapter to use `deepAccess` in places where you are referencing a nested property?